packages/types/src/clerk.ts

```diff
@@ -667,6 +672,20 @@ export type CustomNavigation = (to: string, options?: NavigateOptions) => Promis
 
 export type ClerkThemeOptions = DeepSnakeToCamel<DeepPartial<DisplayThemeJSON>>;
 
+/**
+ * @inline
+ */
+type Experimental = {
```

Comment:
This is a behavior change here since before it was `Autocomplete<>` wrapped.

I don't see why we used it here as the experimental flags should be a set list of properties and not allow for arbitrary things to be passed in